storage/server/frontend.go

```diff
@@ -18,14 +17,13 @@ import (
 
 func (s *server) NVMeSubsystemCreate(ctx context.Context, in *pb.NVMeSubsystemCreateRequest) (*pb.NVMeSubsystemCreateResponse, error) {
 	log.Printf("NVMeSubsystemCreate: Received from client: %v", in)
-	params := BdevMalloCreateParams{
-		Name:      in.GetSubsystem().GetNqn(),
-		BlockSize: 512,
-		NumBlocks: 64,
-		UUID:      uuid.New().String(),
-	}
-	var result BdevAMalloCreateResult
-	err := call("bdev_malloc_create", &params, &result)
+	params := NvmfCreateSubsystemParams{
```

Comment:
Out of curiosity, why is this "Nvmf" but the function name is "NVMe"?

Reply:
because specifically to SPDK they implemented NVMe/PCIe using NVMe/Fabrics adding PCIe as additional fabric (same as DRAM and TCP) ... this is why the name mismatch...